projects/auth0-angular/src/lib/auth.interceptor.spec.ts

```diff
@@ -28,23 +39,25 @@ describe('The Auth HTTP Interceptor', () => {
   let authState: AuthState;
   const testData: Data = { message: 'Hello, world' };
 
-  const assertAuthorizedApiCallTo = (
+  const assertAuthorizedApiCallTo = async (
     url: string,
     done: () => void,
     method = 'get'
   ) => {
     httpClient.request(method, url).subscribe(done);
     flush();
+    await new Promise(process.nextTick);
     req = httpTestingController.expectOne(url);
 
     expect(req.request.headers.get('Authorization')).toBe(
       'Bearer access-token'
     );
   };
 
-  const assertPassThruApiCallTo = (url: string, done: () => void) => {
+  const assertPassThruApiCallTo = async (url: string, done: () => void) => {
     httpClient.get<Data>(url).subscribe(done);
     flush();
+    await new Promise(process.nextTick);
```

Comment:
It looks like Jest handles these Http tests a bit differently, this made it work for all tests, but had to make the function to be async and await it in every test.

Before merging this I will have another look and see if we can avoid this. But for now, this seems to be what solves these tests to run successfully with Jest.